src/models/financialDelinquentPayments.js

```diff
@@ -0,0 +1,70 @@
+import address from 'models/shared/locations/address'
+
+const infractionOptions = [
+  'Alimony',
+  'Judgement',
+  'Lien',
+  'Federal',
+]
+
+const financialDelinquentPayments = {
+  Name: (value, attributes, attributeName, options) => {
+    const { requiredFinancialDelinquentName } = options
+    if (requiredFinancialDelinquentName) {
+      return { presence: true, hasValue: true }
+    }
+    return {}
+  },
+  Infractions: (value, attributes, attributeName, options) => {
+    const { requiredFinancialDelinquentInfraction } = options
+    if (requiredFinancialDelinquentInfraction) {
+      return {
+        presence: true,
+        array: {
+          validator: {
+            presence: true,
+            inclusion: infractionOptions,
+          },
+          length: { minimum: 1 },
+        },
+      }
+    }
+    return {}
+  },
+  AccountNumber: { presence: true, hasValue: true },
+  PropertyType: { presence: true, hasValue: true },
+  Amount: {
+    presence: true,
+    hasValue: {
+      validator: {
+        numericality: {
+          greaterThan: 0,
+        },
+      },
+    },
+  },
+  Reason: { presence: true, hasValue: true },
+  Status: { presence: true, hasValue: true },
+  Date: {
+    presence: true,
+    date: { requireDay: false },
+  },
+  Resolved: (value, attributes) => {
```

Comment:
Section still shows valid if date is left blank.

Reply:
This might be related to the branch not clearing issue. Did you previously have `Not resolved` at any point?